syntax = "proto3";

message CommonRequestByte {
    bytes command = 1;
}